kitsune/wiki/utils.py

```diff
@@ -290,3 +296,70 @@ def build_topics_data(request: HttpRequest, product: Product, topics: list[Topic
         topics_data.append(topic_data)
 
     return topics_data
+
+
+def clean_kb_visited(session: SessionBase, ttl: int = KB_VISITED_DEFAULT_TTL) -> None:
```

Comment:
should this be renamed to something that will indicate that it cleans expired slugs? I was confused a couple of times by the call of this function and I had to check the code to understand that it doesn't clean all slugs

Reply:
I renamed to `remove_expired_from_kb_visited`.